amy/autoemails/models.py

```diff
@@ -222,6 +223,12 @@ def build_email(
             body=body.text,
         )
         msg.attach_alternative(body.html, "text/html")
+        # https://anymail.readthedocs.io/en/stable/sending/templates/#batch-sending-with-merge-data
+        # https://anymail.readthedocs.io/en/stable/sending/templates/#anymail.message.AnymailMessage.merge_data
+        # When set the recipients in the two column each get an individual message
+        msg.merge_data = merge_data
+        if merge_data is not None:
+            assert len(msg.to) <= settings.BULK_EMAIL_LIMIT
```

Comment:
We definitely have more users than 1000. Will there be a mechanism to invoke numerous bulk emails when a new consent comes in?